src/docs/user/references/SchedulerGraphQLAPI.adoc

```diff
@@ -0,0 +1,380 @@
+
+ProActive Scheduler GraphQL API portal can be accessed in your environment using the following link:
+
+++++
+<a class="schedulerGraphQLUrl" href="/scheduling-api/v1/graphiql" target="_blank">Scheduler GraphQL API</a> 
+++++
+
+In general the GraphiQL interface can be reached with `http://your-server:8080/scheduling-api/v1/graphiql`
+
+In order to use it, you must be first connected to one of the ProActive portals. If you are not currently connected or if your session has expired, you will receive a `MissingSessionIdException` when trying to run queries.
+
+This portal allows to test and run GraphQL queries against the scheduler database, to find jobs according to various filters.
+GraphQL queries are read-only, you cannot use this portal to perform stateful operations, such as submitting a job, deleting a job, etc.
+
+The GraphiQL interface is displayed as the following:
+
+image::SchedGraphqlMain.png[align="center"]
+
+You can explore the `Docs` section to see detailed information about all graphql nodes, filters and attributes.
+
+===== GraphQL Basic Usage
+
+The left panel allows to write queries. You can start by entering the query _{ jobs }_, this query will be automatically expanded to match the correct syntax:
+
+[source,graphql]
+----
+{jobs {
+  edges {
+    node {
+      id
+    }
+  }
+}}
+----
+Executing this query will return a json result such as:
+
+[source,json]
+----
+{
+  "data": {
+    "jobs": {
+      "edges": [
+        {
+          "node": {
+            "id": "1"
+          }
+        },
+        {
+          "node": {
+            "id": "2"
+          }
+        },
+        {
+          "node": {
+            "id": "3"
+          }
+        }
+      ]
+    }
+  }
+}
+----
+In this basic example, only the job ids are displayed, it is of course possible to display more information:
+
+[source,graphql]
+----
+{
+  jobs {
+    edges {
+      node {
+        id
+        name
+        owner
+        submittedTime
+        status
+      }
+    }
+  }
+}
+----
+Will return something similar to:
+[source,json]
+----
+{
+  "data": {
+    "jobs": {
+      "edges": [
+        {
+          "node": {
+            "id": "1",
+            "name": "script_groovy",
+            "owner": "admin",
+            "submittedTime": 1589799002016,
+            "status": "FINISHED"
+          }
+        },
+        {
+          "node": {
+            "id": "2",
+            "name": "script_groovy",
+            "owner": "admin",
+            "submittedTime": 1589799356783,
+            "status": "FINISHED"
+          }
+        },
+        {
+          "node": {
+            "id": "3",
+            "name": "Script_Cmd",
+            "owner": "admin",
+            "submittedTime": 1589810586677,
+            "status": "FINISHED"
+          }
+        }
+      ]
+    }
+  }
+}
+----
+
+===== GraphQL Pagination
+GraphQL results are *paginated*, a request can use either of the following attributes to control pagination:
```

Comment:
either of => one of

Reply:
actually, it is one, or two, or three. So either of the is correct. One of means only one attribute can be used.